docs/table-partitioning-bucketing-sort-order.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Table partitioning, bucketing and sorting order"
+slug: /table-partitioning-bucketing-sort-order
+date: 2023-12-25
+keyword: Table Partition Bucket Distribute Sort By
+license: Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Partitioned table
+As well as
+Currently, Gravitino supports the following partitioning strategies:
```

Comment:
What's the meaning of "As well as" and "Currently"?

Reply:
As well as" was left by accident, I have removed it.